### PR TITLE
fix: Improve attribution of measured rates

### PR DESCRIPTION
**Pull request by ThomasGalesloot**
# Pull Request Key Information

## Reason for change

Previously, all emissions at a component were attributed the total measured rate at the component.

## What was changed

This change now divides the measured rate by the number of emissions before attributing a rate to each emission.

## Intended Purpose

Improve emissions attribution in modelling

## Level of version change required

N/A

## Testing Completed

Added new unit test and manual testing

## Target Issue

N/A

## Additional Information

N/A

### CODE REVIEW

File: LDAR_Sim/testing/unit_testing/test_virtual_world/test_equipment/test_tag_emissions.py
Comment (SEAJang) [inline LDAR_Sim/testing/unit_testing/test_virtual_world/test_equipment/test_tag_emissions.py:1]: Please add doc-string/MIT license.

Comment (SEAJang) [inline LDAR_Sim/testing/unit_testing/test_virtual_world/test_equipment/test_tag_emissions.py:1]: Are the mocked objects something that could be reused for other unit tests? If so, they should be moved to a file of their own.

**Review by SEAJang — APPROVED** [review]
Great job!

File: LDAR_Sim/testing/unit_testing/test_virtual_world/test_equipment/test_tag_emissions.py
Comment (ThomasGalesloot) [reply LDAR_Sim/testing/unit_testing/test_virtual_world/test_equipment/test_tag_emissions.py:1]: Sort of, not really. Will take this offline.

**Pull request merged**